src/tree/SubscriptionTreeItem.ts

```diff
@@ -187,9 +187,6 @@ export class SubscriptionTreeItem extends SubscriptionTreeItemBase {
             if (focusedGroup) {
                 return focusedGroup;
             }
-
-            // if we can't find it, erase the current setting, return current cached tree
-            await ext.context.workspaceState.update('focusedGroup', '');
```

Comment:
Clearing this on the first call to `tryGetFocusedGroup` breaks the feature. It behaves fine if we never clear it.